web/src/components/mainPageCarousel/style.css

```diff
@@ -0,0 +1,38 @@
+.carousel__item--container {
+    position: relative;
+    width: 50%;
+}
+
+.carousel__img {
+    margin-top: 40px;
+    opacity: 1;
+    display: block;
+    max-width: 350px;
+    height: auto;
+    transition: 0.5s ease;
+    backface-visibility: hidden;
+}
+
+.carousel__txt--container {
+    transition: 0.5s ease;
+    opacity: 0;
+    position: absolute;
+    top: 50%;
+    left: 80%;
+    transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    text-align: center;
+}
+
+.carousel__item--container:hover .carousel__img {
+    opacity: 0.3;
+}
+
+.carousel__item--container:hover .carousel__txt--container {
+    opacity: 1;
+}
+
+.carousel__txt {
+    font-size: 20px;
+    color: #04aa6d;
+}

```

Comment:
hmmm, I would reccomend to change selector names. it can have some conflicts with carousel styling